assets/style_codebases.scss

```diff
@@ -0,0 +1,348 @@
+---
+---
+
+@import url('https://fonts.googleapis.com/css?family=Muli:300');
+
+:root {
+  --fpc-yellow: #FFF500;
+  --fpc-purple: #5B57CA;
+  --fpc-purple-a: #5B57CA35;
+  --fpc-blue: #56CCF2;
+  --fpc-blue-g: #0FBBF2;
+  --fpc-blue-a: #56CCF235;
+  --fpc-peach: #FDBBAB;
+  --fpc-peach-a: #FDBBAB35;
+  --fpc-green: #00BA8D;
+  --fpc-green-a: #00BA8D35;
+  --fpc-grey: #E3E3E3;
+  --fpc-grey-a: #E3E3E3CC;
+  --fpc-grey-a2: #E3E3E364;
+}
+
+$breakpoints: (
+  'mobile': ( max-width:  640px ),
+  'small':  ( max-width:  767px ),
+  'medium': ( max-width:  992px ),
+  'large':  ( max-width: 1200px ),
+  'max':    ( min-width: 1600px )
+) !default;
+
+// Mixin to manage responsive breakpoints
+// @author Hugo Giraudel
+// @param {String} $breakpoint - Breakpoint name
+// @require $breakpoints
+@mixin respond-to($breakpoint) {
+  // If the key exists in the map
+  @if map-has-key($breakpoints, $breakpoint) {
+    // Prints a media query based on the value
+    @media #{inspect(map-get($breakpoints, $breakpoint))} {
+      @content;
+    }
+  }
+  // If the key doesn't exist in the map
+  @else {
+    @warn "Unfortunately, no value could be retrieved from `#{$breakpoint}`. "
+      + "Available breakpoints are: #{map-keys($breakpoints)}.";
+  }
+}
+
+
+// LIVE CSS OVERRIDES - HOW BEST TO RECONCILE?
+
+p, h1, h2, h3, h4, h5, ul, pre, blockquote, .highlight {
+  max-width: none;
```

Comment:
Careful: this can cause blocks of text to have excessively long line length.

Reply:
Captured in https://github.com/publiccodenet/publiccode.net/issues/153